.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    "env": {
+        "node": true,
+        "commonjs": true,
+        "es2021": true,
+	"mocha": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12
+    },
+    "rules": {
```

Comment:
Where do those rules come from? Is this some convention?

Reply:
https://eslint.org/docs/latest/rules/

Reply:
I mean why we settled on:
```
 "quotes": ["error", "double"],
  "semi": ["error", "never"]
```
seems the opposite of what Google Style guide for example suggests:
https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

I don't think we have a company-wide conventions for this.